resources/guest_configs/patches/0003-enable-ftrace.patch

```diff
@@ -0,0 +1,91 @@
++CONFIG_FTRACE=y
+ # CONFIG_DMA_API_DEBUG is not set
+ # CONFIG_KUNIT is not set
+ 
+diff --git a/resources/guest_configs/microvm-kernel-ci-aarch64-5.10.config b/resources/guest_configs/microvm-kernel-ci-aarch64-5.10.config
+index ac44904c1..0b72b4eee 100644
+--- a/resources/guest_configs/microvm-kernel-ci-aarch64-5.10.config
++++ b/resources/guest_configs/microvm-kernel-ci-aarch64-5.10.config
+@@ -3089,7 +3089,7 @@ CONFIG_HAVE_FTRACE_MCOUNT_RECORD=y
+ CONFIG_HAVE_SYSCALL_TRACEPOINTS=y
+ CONFIG_HAVE_C_RECORDMCOUNT=y
+ CONFIG_TRACING_SUPPORT=y
+-# CONFIG_FTRACE is not set
++CONFIG_FTRACE=y
+ # CONFIG_SAMPLES is not set
+ CONFIG_ARCH_HAS_DEVMEM_IS_ALLOWED=y
+ # CONFIG_STRICT_DEVMEM is not set
+diff --git a/resources/guest_configs/microvm-kernel-ci-aarch64-6.1.config b/resources/guest_configs/microvm-kernel-ci-aarch64-6.1.config
+index 26b87a658..257e0da14 100644
+--- a/resources/guest_configs/microvm-kernel-ci-aarch64-6.1.config
++++ b/resources/guest_configs/microvm-kernel-ci-aarch64-6.1.config
+@@ -3309,7 +3309,7 @@ CONFIG_HAVE_FTRACE_MCOUNT_RECORD=y
+ CONFIG_HAVE_SYSCALL_TRACEPOINTS=y
+ CONFIG_HAVE_C_RECORDMCOUNT=y
+ CONFIG_TRACING_SUPPORT=y
+-# CONFIG_FTRACE is not set
++CONFIG_FTRACE=y
+ # CONFIG_SAMPLES is not set
+ # CONFIG_STRICT_DEVMEM is not set
+ 
+diff --git a/resources/guest_configs/microvm-kernel-ci-x86_64-4.14.config b/resources/guest_configs/microvm-kernel-ci-x86_64-4.14.config
+index ee6df5ffc..3b7e8f10c 100644
+--- a/resources/guest_configs/microvm-kernel-ci-x86_64-4.14.config
++++ b/resources/guest_configs/microvm-kernel-ci-x86_64-4.14.config
+@@ -2694,7 +2694,7 @@ CONFIG_HAVE_SYSCALL_TRACEPOINTS=y
+ CONFIG_HAVE_FENTRY=y
+ CONFIG_HAVE_C_RECORDMCOUNT=y
+ CONFIG_TRACING_SUPPORT=y
+-# CONFIG_FTRACE is not set
++CONFIG_FTRACE=y
+ # CONFIG_PROVIDE_OHCI1394_DMA_INIT is not set
+ # CONFIG_DMA_API_DEBUG is not set
+ # CONFIG_KUNIT is not set
+diff --git a/resources/guest_configs/microvm-kernel-ci-x86_64-5.10-no-acpi.config b/resources/guest_configs/microvm-kernel-ci-x86_64-5.10-no-acpi.config
+index b87fb3e44..d8e72db90 100644
+--- a/resources/guest_configs/microvm-kernel-ci-x86_64-5.10-no-acpi.config
++++ b/resources/guest_configs/microvm-kernel-ci-x86_64-5.10-no-acpi.config
+@@ -2905,7 +2905,7 @@ CONFIG_HAVE_SYSCALL_TRACEPOINTS=y
+ CONFIG_HAVE_FENTRY=y
+ CONFIG_HAVE_C_RECORDMCOUNT=y
+ CONFIG_TRACING_SUPPORT=y
+-# CONFIG_FTRACE is not set
++CONFIG_FTRACE=y
+ # CONFIG_SAMPLES is not set
+ CONFIG_ARCH_HAS_DEVMEM_IS_ALLOWED=y
+ CONFIG_STRICT_DEVMEM=y
+diff --git a/resources/guest_configs/microvm-kernel-ci-x86_64-5.10.config b/resources/guest_configs/microvm-kernel-ci-x86_64-5.10.config
+index 09461c178..2bf2c569a 100644
+--- a/resources/guest_configs/microvm-kernel-ci-x86_64-5.10.config
++++ b/resources/guest_configs/microvm-kernel-ci-x86_64-5.10.config
+@@ -2987,7 +2987,7 @@ CONFIG_HAVE_SYSCALL_TRACEPOINTS=y
+ CONFIG_HAVE_FENTRY=y
+ CONFIG_HAVE_C_RECORDMCOUNT=y
+ CONFIG_TRACING_SUPPORT=y
+-# CONFIG_FTRACE is not set
++CONFIG_FTRACE=y
```

Comment:
I think it needs more options enabled: https://stackoverflow.com/questions/41238386/how-to-enable-or-configure-ftrace-module

Reply:
I thought just enabling `ftrace` will enable at least `function_tracer` and `function_graph_tracer`. Anyway, enabled other options as well.